### DISCUSSION TITLE
Hey! maybe a naive question, but I dont seem to get the concept of confirm.

### DISCUSSION THREAD

**Discussion by Meeshalk** [bavix/laravel-wallet#935] [Q&A]
What exactly happens when we confirm?
What could we confirming? 

There is no explanation in the documentation and I have searched everywhere about its useage. No luck 😭 

Any help is appreciate.
Thanks

**Comment by rez1dent3** [comment]
Hello. Confirmation of an unconfirmed transaction.
It works like this:
1. A lock is placed and a transaction is opened in the database;
2. The selected transaction is confirmed;
3. Balance changes are applied;
4. The lock is removed and the database transaction is confirmed;

Docs: https://bavix.github.io/laravel-wallet/#/confirm
